routes/products.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const router = express.Router();
+const productDB = require('../db/products.js');
+const database = productDB.getProducts().products;
+const middleware = require('../middleware/validate');
+
+/************************
+ *  GET
+************************/
+
+router.get('/', (req, res) => {
+  const products = productDB.getProducts();
+  if (products.length < 1) {
+    products.message = "No products in our database";
+  } else {
+    products.message = "";
+  }
+
+  res.status(200);
+  return res.render('./products/index', products);
+});
+
+router.get('/new', (req, res) => {
+  res.status(200);
+  return res.render('./products/new', {});
+});
+
+router.get('/:id', middleware.validator, (req, res) => {
+  const params = req.params;
+  const productIndex = productDB.findProduct(params.id);
```

Comment:
This should use a function to get a single product, please create one if you don't have one.